WordPress/gutenberg
# Run script loader test

## What?

This PR teaches PHPUnit to run script loader tests.

## Why?

They were not being executed.

## How?

By adding the `-test.php` suffix.

## Testing Instructions

Locally, make sure they are executed: `npm run test:unit:php -- --filter WP_Script_Loader_Test`.
